tests/test_libinteract.py

```diff
@@ -78,6 +161,60 @@ def ref_hc(ref_hc_file):
     with open(ref_hc_file) as fh:
         return fh.readlines()
 
+# inter chain table files
```

Comment:
any reason we're not loading "intra_B" datasets for SB and HC?

Reply:
There were no intra B contacts in the twochains file.

Reply:
Ok. We should also ask ourselves what we want to do in this case - does the user still expect an output file with no interactions?

Reply:
Sorry, I missed this one. I log a warning when no intrachain or interchain contacts are found. Is this sufficient? I assume the user won't expect a file if the warning tells them no contacts exist.

Reply:
yep that's fine